### ISSUE TITLE
spaghetti.software domain name

### ISSUE THREAD

**Issue by JoelOsler** [Heggemm/Heggemm.github.io#1]
Hi, so a few weeks ago I was going to purchase a domain name for my portfolio, and it seems that you have already claimed it. I already had a entire template created and my website was basically ready to be live, but I was upset as you purchased the domain name that I was planning on getting a few weeks ago. Do you think I might be able to buy it off of you? I have a backup domain name for now, "spaghetti.gq" but my setup doesn't seem right without the signature name that's literally part of what I make. I'm a developer whom makes software, which contains "spaghetti code" but it works really well. (Thats kind the whole joke of the domain name) I got inspiration for my domain name from a near friend on Twitter, @BurritoSoftware and I thought that was a cool name, and he also owns "burrito.software".

TLDR: Can I take this domain off of your hands for some scrumptious money? I will only need the domain for about 0.5-1 year, and then it will be available to purchase again by the public.

Can ya help a fellow student out?

-Joel
Developer of THE original Spaghetti Software 🍝, Project Homebase 🏠, Etcetera Development 🧑‍💻

**Comment by Heggemm** [comment]
Hello!

It seems we had the same idea, sorry to hear that, but I am also a programmer and I pride myself on writing especially good spaghett code haha! 
I am very hesistant to give it up as it is a pretty cool domain name, and I was going to make my website on it in the future, but I don't really have any immediate plans for it so I'm going to give it some thought 🙂 
How much scrumptious money were you thinking haha?

Martin K. J. Heggem
--
Master's Student | Department of Informatics | UiO

**Comment by JoelOsler** [comment]
Ok! I'll let you keep it, because I might have something different in mind for now...
I'm also curious on what your going to make so I might take a peek now and then 😄
